lib/src/value/calculation.dart

```diff
@@ -223,6 +605,83 @@ class SassCalculation extends Value {
   /// simplification.
   SassCalculation._(this.name, this.arguments);
 
+  // Returns the value coerced to numbers units.
```

Comment:
Nit:

```suggestion
  // Returns [value] coerced to [number]'s units.
```

When referring to a function's arguments in Dart, you generally use the argument name surrounded by `[]`. This tells DartDoc to render it as monospace, and indicates to the reader that you're referring to a specific code value rather than a general prose term.